starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1374,22 +1374,37 @@ sub check_star_catalog {
 	# Bad Pixels ACA-025
         my @close_pixels;
         my @dr;
-	if ($type ne 'ACQ' and $c->{"GS_PASS$i"} =~ /^1|\s+|g[1-2]/) {
+        my @dz;
+        my @dy;
+	if ($type ne 'MON' and $type ne 'FID'){
 	    foreach my $pixel (@bad_pixels) {
 		my $dy = abs($yag-$pixel->{yag});
 		my $dz = abs($zag-$pixel->{zag});
 		my $dr = sqrt($dy**2 + $dz**2);
-		next unless ( $dz < $dither+25 and $dy < $dither+25 );
+		next unless ( $dz < $c->{"HALFW$i"} and $dy < $c->{"HALFW$i"});
 		push @close_pixels, sprintf("%3d, %3d, %3d\n", $dy, $dz, $dr);
+                push @dy, $dy;
+                push @dz, $dz;
 		push @dr, $dr;
-	    }   
+	    }
 	    if ( @close_pixels > 0 ) {
 		my ($closest) = sort { $dr[$a] <=> $dr[$b] } (0 .. $#dr);
```

Comment:
This still just looks for the closest pixel (no change from previous behavior).  Do we care if an acq box has more than one bad pixel?